llvm/lib/CodeGen/InlineSpiller.cpp

```diff
@@ -167,8 +167,8 @@ class InlineSpiller : public Spiller {
   // All registers to spill to StackSlot, including the main register.
   SmallVector<Register, 8> RegsToSpill;
 
-  // All registers that were replaced by the spiller. This includes registers
-  // that were rematerialized; rematted regs are removed from RegsToSpill.
+  // All registers that were replaced by the spiller through some other method,
+  // e.g. rematerialization.
   SmallVector<Register, 8> RegsReplaced;
```

Comment:
Maybe could make this one set, if there's another way to tell which were spilled and which were just rematerialized

Reply:
I started implementing this change, but it really didn't turn out that well.